simtools/simtel/simtel_config_reader.py

```diff
@@ -159,27 +168,57 @@ def export_parameter_dict_to_json(self, file_name, dict_to_write):
     def compare_simtel_config_with_schema(self):
         """
         Compare limits and defaults reported by simtel_array with schema
-        (for debugging purposes; simple printing).
+        (for debugging purposes; simple printing). Check for differences
+        in 'default' and 'limits' entries.
 
         """
 
-        self._logger.info(
-            f"Comparing simtel_array configuration with schema for {self.parameter_name} "
-            f"(sim_telarray: {self.simtel_parameter_name})"
-        )
+        for data_type in ["default", "limits"]:
+            _from_simtel = self.parameter_dict.get(data_type)
+            # ignore limits checks for boolean
+            if data_type == "limits" and self.parameter_dict.get("type") == "bool":
+                continue
+            try:
+                if data_type == "limits":
+                    _from_schema = [
+                        self.schema_dict["data"][0]["allowed_range"].get("min"),
+                        self.schema_dict["data"][0]["allowed_range"].get("max"),
+                    ]
+                    _from_schema = _from_schema[0] if _from_schema[1] is None else _from_schema
+                else:
+                    if len(self.schema_dict["data"]) == 1:
+                        _from_schema = self.schema_dict["data"][0]["default"]
+                    else:
+                        _from_schema = [data.get("default") for data in self.schema_dict["data"]]
+            except (KeyError, IndexError):
+                _from_schema = None
+
+            if isinstance(_from_schema, list):
+                _from_schema = np.array(_from_schema, dtype=np.dtype(self.parameter_dict["type"]))
 
-        self._logger.info("Limits:")
-        self._logger.info(f"  from simtel: {self.parameter_dict.get('limits')}")
-        try:
-            self._logger.info(f"  from schema: {self.schema_dict['data'][0]['allowed_range']})")
-        except (KeyError, IndexError):
-            self._logger.info("  from schema: None")
-        self._logger.info("Defaults:")
-        self._logger.info(f"  from simtel: {self.parameter_dict.get('default')}")
-        try:
-            self._logger.info(f"  from schema: {self.schema_dict['data'][0]['default']}")
-        except (KeyError, IndexError):
-            self._logger.info("  from schema: None")
+            try:
+                if (
+                    not isinstance(_from_schema, (list, np.ndarray))
+                    and _from_simtel == _from_schema
+                ):
+                    self._logger.debug(f"Values for {data_type} match")
+                    continue
+            except ValueError:
+                pass
```

Comment:
No raise here or warning?

Reply:
Ok I see now that you add warnings later.

Reply:
Good, I think this is resolved.